windows/src/engine/keyman32/Keyman32.cpp

```diff
@@ -755,7 +764,7 @@ void LoadBaseLayoutSettings() {   // I4552   // I4583
 
     Globals::SetBaseKeyboardFlags(underlyingLayout, reg->ReadInteger(REGSZ_SimulateAltGr), !reg->ValueExists(REGSZ_DeadkeyConversionMode) || reg->ReadInteger(REGSZ_DeadkeyConversionMode));
 	} else {
-    Globals::SetBaseKeyboardFlags("", 0, 0);
+    Globals::SetBaseKeyboardFlags("", 0, 1);
```

Comment:
This is unlikely to ever be called, but just noting that the default for deadkeyconversionmode should be TRUE, not FALSE